end_to_end/fraud_detection/pipeline-e2e.ipynb

```diff
@@ -4,32 +4,37 @@
    "cell_type": "markdown",
```

Comment:
<p>Is there a reason background does not show up with reqd. formatting?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Ah, that is a ReviewNB problem - you can see it is properly formatted in https://github.com/atqy/amazon-sagemaker-examples/blob/atqy/refactor-fraud-detection/end_to_end/fraud_detection/pipeline-e2e.ipynb</p>